examples/PDP.ipynb

```diff
@@ -0,0 +1,679 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e4c8a080",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "SLF4J: Failed to load class \"org.slf4j.impl.StaticLoggerBinder\".\n",
+      "SLF4J: Defaulting to no-operation (NOP) logger implementation\n",
+      "SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from trustyai.model import feature, PredictionInput\n",
+    "from trustyai.utils import TestModels\n",
+    "from trustyai.explainers.pdp import PDPExplainer, PredictionProviderMetadata\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from sklearn.datasets import make_classification"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Can you add some text/description of each (or sort of) step? Just to briefly explain what is going on

Reply:
sure, will do.